code/game/objects/items/melee/misc.dm

```diff
@@ -76,17 +76,11 @@
 		final_block_chance = 0 //Don't bring a sword to a gunfight
 	return ..()
 
-/obj/item/melee/sabre/on_exit_storage(obj/item/storage/S)
-	..()
-	var/obj/item/storage/belt/sabre/B = S
-	if(istype(B))
-		playsound(B, 'sound/items/unsheath.ogg', 25, 1)
```

Comment:
```
/obj/item/melee/sabre/on_exit_storage(datum/component/storage/concrete/S)
	..()
	var/obj/item/storage/belt/sabre/B = S.real_location()
	if(istype(B))
		playsound(B, 'sound/items/unsheath.ogg', 25, 1)
```

Reply:
I deleted the parent call because the parent call was just a stub.